src/graph/query_graph.h

```diff
@@ -33,6 +33,9 @@ QueryGraph *QueryGraph_New(uint node_cap, uint edge_cap);
 /* Adds a new node to the graph */
 void QueryGraph_AddNode(QueryGraph *g, QGNode *n);
 
+/* Set the label and associated data on a QueryGraph node. */
```

Comment:
I think `Set the label on a QueryGraph node.` is enough, it is hard to tell what is the `associated data`